app/test/frontend/golden/pkg_admin_page.html

```diff
@@ -50,8 +50,17 @@
         </form>
       </div>
       <div class="site-header-nav scroll-container">
-        <div class="nav-login-container">
-          <button id="-account-login" class="nav-main-button link">Sign in</button>
+        <div id="-account-profile" class="nav-container nav-my-container hoverable">
+          <button class="nav-main-button">My pub.dev</button>
```

Comment:
Is this intentional? just asking.

Reply:
Yes, the template test was rendering an admin page, and the rendering reflected the signed-in user. Fixed that in the templates_test.dart + golden update.